tests/tape_tests/test_ensemble.py

```diff
@@ -832,6 +832,38 @@ def test_persist(dask_client):
     assert new_graph_size < old_graph_size
 
 
+@pytest.mark.parametrize(
+    "data_fixture",
+    [
+        "parquet_ensemble_with_divisions",
+        "parquet_ensemble_without_client",
```

Comment:
Despite the codecov complaint about the missing line, the "parquet_ensemble_without_client" here should be exercising the client=False case